wifi_manager/wifi_manager.py

```
@@ -589,7 +776,7 @@ def _check_and_notify_connection_state(cls):
                         config = cls.wlan().config('ssid')
                         if config:
                             ssid = config
-                    except:
+                    except Exception:
                         pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)